gobblin-core/src/main/java/gobblin/converter/string/TextToStringConverter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.converter.string;
+
+import org.apache.avro.Schema;
+import org.apache.hadoop.io.Text;
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.converter.Converter;
+import gobblin.converter.DataConversionException;
+import gobblin.converter.SchemaConversionException;
+import gobblin.converter.SingleRecordIterable;
+
+
+/**
+ * An implementation of {@link Converter} that converts input records of type {@link Text} to strings.
+ *
+ * @author ynli
+ */
+@SuppressWarnings("unused")
+public class TextToStringConverter extends Converter<Schema, Schema, Text, String> {
```

Comment:
Good catch. The schema can be any type here since it's not used. I will change it to `Object`.